filament/src/fg/ResourceAllocator.cpp

```diff
@@ -113,6 +113,15 @@ backend::TextureHandle ResourceAllocator::createTexture(const char* name,
         backend::SamplerType target, uint8_t levels,
         backend::TextureFormat format, uint8_t samples, uint32_t width, uint32_t height,
         uint32_t depth, backend::TextureUsage usage) noexcept {
+
+    if (!(usage & TextureUsage::SAMPLEABLE)) {
```

Comment:
Maybe do 16x16 because of 720p.

Reply:
Also worried that this will take a slow path (e.g. disable UBWC) on some drivers.